### ISSUE TITLE
[BUG] COPY CSV file with delimiter runs in syntax error

### ISSUE THREAD

**Issue by tantweiler** [cweijan/vscode-mysql#1219]
- Database type and version: PostgreSQL 16.3

I use the following query to import data from a CSV file with semicolon delimiter:

```
COPY test_csv (value1, value2, value3, value4, value5, value6) FROM 'test.csv' DELIMITER ';' CSV HEADER;
```
Error message:
```
invalid input syntax for type integer: "10100;Joe;1;0.5;Denver;Texas"
```
The syntax works in psql cli tool and beekeeper studio without any issues. If I change the delimiter in the CSV file to comma separated, then it works! Since the same query works in other tools I think this is a bug here with the vscode-database-client and semicolon delimiter within the COPY command.

**Comment by BusraEcemSakar** [comment]
you have null values in your data, that is why you have a multiple ;, can you try this sql? 

copy test_csv (value1, value2, value3, value4, value5, value6) from 'test.csv' with (FORMAT csv, HEADER, NULL '');` 

and Value4 variable type should be defined to handle null values.

**Comment by tantweiler** [comment]
@BusraEcemSakar 
Same problem with that query and I also tried that before already but didn't mention it. Again, the query works with psql cli tool and beekeeper-studio. So the query cannot be wrong in my opinion.

**Comment by cweijan** [comment]
Fixed in version 7.5.7.

**Comment by tantweiler** [comment]
@cweijan Thank you very much for the fix! It works as expected now.
